app/views/core/CreateAccountModal/CreateAccountModal.coffee

```diff
@@ -130,7 +130,10 @@ module.exports = class CreateAccountModal extends ModalView
           @signupState.set { screen: 'segment-check' }
       'signup': ->
         if @signupState.get('path') is 'student'
-          @signupState.set { screen: 'extras', accountCreated: true }
+          if me.showHeroAndInventoryModals() # if classroomItems feature is on, don't display hero selection option, go to confirmation directly
```

Comment:
Let's add a me method that applies to this context, so we don't need a comment referencing low level feature settings.